styles/electrified.json

```diff
@@ -20,7 +21,8 @@
 				"coordinates": [[10,50],[80,50]],
 				"properties": {
 					"railway":"rail",
-					"electrified":"no"
+					"electrified":"no",
+					"usage":"main"
```

Comment:
Just a small nitpick: if you move the new line right after the ```"railway":"rail"``` then this will only change one line for every hunk but the first one.

Reply:
Sure, that would be cleaner. Do I make a new PR then?

Reply:
Should be good now?